sequence.go

```diff
@@ -654,28 +659,36 @@ func findSequencesOnDisk(path string, opts ...FileOption) (FileSequences, error)
 
 		frame, _ := strconv.Atoi(match[2])
 		key := [2]string{match[1], match[3]}
-		frames, ok := seqs[key]
+		seq, ok := seqs[key]
+		frameWidth := len(match[2])
 		if !ok {
-			frames = []int{frame}
-			padMap[key] = padder.PaddingChars(len(match[2]))
+			seq = seqInfo{
+				Frames:   []int{frame},
+				Padding:  padder.PaddingChars(frameWidth),
+				MinWidth: frameWidth,
+			}
 		} else {
-			frames = append(frames, frame)
+			seq.Frames = append(seq.Frames, frame)
+			if seq.MinWidth > frameWidth {
+				seq.MinWidth = frameWidth
+				seq.Padding = padder.PaddingChars(frameWidth)
+			}
 		}
-		seqs[key] = frames
+		seqs[key] = seq
```

Comment:
Hmm, I think there will be a problem because MinWidth will be initialized as 0 right? Then it would never > frameWidth